### DISCUSSION TITLE
How to use database here, which database should I use?

### DISCUSSION THREAD

**Discussion by Tushar-Beladiya** [excalidraw/excalidraw#4276] [Q&A]
I want to store the diagram or shapes inside the database so If I will open this project on another PC anywhere so I can fetch old data from database and display to user what he has created in past.

Thank you

**Comment by sohailamjad256** [comment]
yes ..you can do just you need the send the initial data file to api ..and through the get call you will get the the json data and that and can show that

**Reply by sohailamjad256** [reply]
any data base you can use ..while saving scene you need call post call and inside that you need to send initial data json
